validate-charts/dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ghcr.io/openzipkin/alpine:3.12.1
+
+# Use latest recommended version here
+ARG HELM_VERSION=3.3.0
```

Comment:
Does it make sense to upgrade helm version to latest as long as we're rolling this out (3.4.2)?

Reply:
I am updating it considering there won't be any compatibility issue as such.